### PR TITLE
Add TOML config file loader

### PR DESCRIPTION
**Pull request by ykanda**
I tried to make the loader of TOML files in issue #235
It uses the https://github.com/BurntSushi/toml to implement
The same test as the YAML loader has path

### CODE REVIEW

File: altsrc/toml_command_test.go [hunk 1]
```diff
@@ -0,0 +1,310 @@
+// Disabling building of tom support in cases where golang is 1.0 or 1.1
```
Comment (meatballhat) [inline altsrc/toml_command_test.go]: This should be "toml support", yes?

**Comment by meatballhat** [comment]
LGTM :+1: and I'd like someone more familiar with the guts of ./altsrc to review, too :smile_cat:

**Comment by meatballhat** [comment]
@roboll @ChrisPRobinson :wave: 😺

File: altsrc/toml_file_loader.go [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+import (
+	"fmt"
+	"reflect"
+
+	"github.com/BurntSushi/toml"
+	"github.com/urfave/cli"
+)
+
+type TomlMap struct {
```
Comment (ChrisPRobinson) [inline altsrc/toml_file_loader.go]: Is it required to have this type be public? It seems that its just used internally and its not needed to be public, but not sure if the underlying deserializer requires a public container type

**Comment by ChrisPRobinson** [comment]
I would update Consider updating front page markup to indicate that now it supports TOML too

**Comment by ChrisPRobinson** [comment]
👍 Looks good.

File: altsrc/toml_command_test.go (hunk 1 above)
Comment (ykanda) [inline altsrc/toml_command_test.go]: Yes, I miss it.
I Fix the typo.

File: altsrc/toml_file_loader.go (hunk 2 above)
Comment (ykanda) [inline altsrc/toml_file_loader.go]: It's exactly as you say.
Fixed the name of the type to tomlMap.

**Comment by jszwedko** [comment]
Added notes to README and CHANGELOG in https://github.com/urfave/cli/commit/b616f6088660d2eaa33739718f0583f8d467a178

Thanks @ykanda and @ChrisPRobinson !

**Pull request merged**